src/test/java/InfrastructureManager/AdvantEdge/AdvantEdgeClientTests.java

```diff
@@ -53,6 +56,21 @@ public void deployScenarioRequestTest() {
         verify(postRequestedFor(urlEqualTo(path)));
     }
 
+    @Test
+    public void updateNetworkCharacteristics() throws IOException {
+        String sandboxName = "test";
+        String path = "/" + sandboxName + "/sandbox-ctrl/v1/events/NETWORK-CHARACTERISTICS-UPDATE";
+        String jsonTestPath = "src/test/resources/AdvantEdge/network-update-test.json";
+        client.out("advantEdge networkUpdate " + sandboxName + " fog-1 FOG 10 10 50 1 0");
+
+        ObjectMapper mapper = new ObjectMapper();
+
+        String jsonString = mapper.readValue(Paths.get(jsonTestPath).toFile(), String.class);
```

Comment:
the test fails here while trying to get the file, I assume it doesn't fail locally because the file is visible, but fails on GitHub because the file is not visible with the given path.